### PR TITLE
[14.0][MIG] stock_picking_supplier_ref: Migration to 14.0

### PR DESCRIPTION
**Pull request by Reyes4711-S73**
Standard migration to 14.0

### CODE REVIEW

File: stock_picking_supplier_ref/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds the field supplier reference in pickings.
```
Comment (flotho) [inline stock_picking_supplier_ref/readme/DESCRIPTION.rst]: Which reference are we talking about

File: stock_picking_supplier_ref/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2017 Trey, Kilobytes de Soluciones
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    reference = fields.Char(string="Supplier Reference")
```
Comment (flotho) [inline stock_picking_supplier_ref/models/stock_picking.py]: IMHO reference is too more "open" definition. I mean, are we talking about the reference of the supplier_reference available in the PO, are we talking about the picking reference of the supplier ?
I would recommend to make some small changes to better specify the use case.

**Review by flotho — CHANGES_REQUESTED** [review]
Hi,

Thanks for your work.
I think that small changes could easily improve your contribution.

Regards

**Comment by Reyes4711-S73** [comment]
@flotho In this PR we are doing only the standard migration of this module, not doing some improvements. To do this, we need to create another PR with all the improvements to do.
Besides, changing the name of this field maybe do some problems with people are using this module. In the PR of improvements, you must do a script which does the change of the name of field to the new name.

**Comment by rousseldenis** [comment]
> @flotho In this PR we are doing only the standard migration of this module, not doing some improvements. To do this, we need to create another PR with all the improvements to do. Besides, changing the name of this field maybe do some problems with people are using this module. In the PR of improvements, you must do a script which does the change of the name of field to the new name.

@Reyes4711-S73 You can do for sure changes in this PR. For reference name, I agree with @flotho. To support existing installations, you should provide a migration script.

**Comment by Reyes4711-S73** [comment]
@rousseldenis @flotho In a few days I will do this changes

**Comment by Reyes4711-S73** [comment]
@rousseldenis and @flotho Added suggested improvements. If is all correct, could you merge this PR?

**Review by rousseldenis — APPROVED** [review]

**Review by hildickethan — APPROVED** [review]

**Comment by Reyes4711-S73** [comment]
@rousseldenis Could you merge this PR?

**Comment by rousseldenis** [comment]
@flotho Could you update your review?

@Reyes4711-S73 Could you squash your two last commits (as the add of openupgradelib is linked to the improvement)?

**Review by flotho — APPROVED** [review]
LGTM, thanks for the update

**Comment by Reyes4711-S73** [comment]
@rousseldenis Squashed. Please, could you merge this PR now?

**Review by flotho — APPROVED** [review]

**Review by ferran-S73 — APPROVED** [review]

**Pull request merged**